test_schemas/en/test_multiple_checkbox_answers.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
I don't think this schema is needed as checkbox as one of multiple answers is tested in the `test_multiple_answers` schema.

Reply:
I only added this because it mentioned it on the card - guess i'm ok to remove the newly added functional test as well then?